components/esp_websocket_client/examples/autobahn-testsuite/scripts/quick_test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
We could simplify this by taking the config file as input and use it to drive all configuration. This could provide us the capability of selecting which configuration to run for each scenario without changing the config for CI.